.sass-lint.yml

```diff
@@ -0,0 +1,182 @@
+# sass-lint config generated by make-sass-lint-config v0.1.2
+#
+# The following scss-lint Linters are not yet supported by sass-lint:
```

Comment:
No, I don't think so. I pulled this as a default config from somewhere and it doesn't include things where the default was preferred. Rules are here:

https://github.com/sasstools/sass-lint/tree/master/docs/rules